OpenMarket/OpenMarket/View Controller/EditProductViewController.swift

```diff
@@ -0,0 +1,158 @@
+//
+//  PostProductViewController.swift
+//  OpenMarket
+//
+//  Created by 서녕 on 2022/01/21.
+//
+
+import UIKit
+
+class EditProductViewController: UIViewController, UITextFieldDelegate, UITextViewDelegate {
+    @IBOutlet weak var postImageListCollectionView: UICollectionView!
+    @IBOutlet weak var postImageButton: UIButton!
+    @IBOutlet weak var productNameTextField: UITextField!
+    @IBOutlet weak var productPriceTextField: UITextField!
+    @IBOutlet weak var currencySwitchController: UISegmentedControl!
+    @IBOutlet weak var discountedPriceTextField: UITextField!
+    @IBOutlet weak var productStockTextField: UITextField!
+    @IBOutlet weak var productNavigationBar: UINavigationItem!
+    @IBOutlet weak var productDescription: UITextView!
+    let imagePickerController = UIImagePickerController()
+    var tempPostImage: [UIImage] = []
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        postImageListCollectionView.delegate = self
+        postImageListCollectionView.dataSource = self
+        let flowLayout = UICollectionViewFlowLayout()
+        flowLayout.scrollDirection = .horizontal
+        flowLayout.minimumLineSpacing = 0
+        postImageListCollectionView.collectionViewLayout = flowLayout
+        postImageListCollectionView.isPagingEnabled = true
+        
+        imagePickerController.delegate = self
+        placeholderSetting()
+    }
+    
+    func placeholderSetting() {
+        productNameTextField.delegate = self
+        productPriceTextField.delegate = self
+        discountedPriceTextField.delegate = self
+        productStockTextField.delegate = self
+        productNameTextField.placeholder = "상품명"
+        productPriceTextField.placeholder = "상품가격"
+        discountedPriceTextField.placeholder = "할인금액"
+        productStockTextField.placeholder = "재고수량"
+    }
+    
+    @IBAction func hitDoneButton(_ sender: Any) {
+        let paramData: [String:String] = ["name" : productNameTextField.text ?? "",
```

Comment:
일반적인 컨벤션을 맞춘다면 이렇게 공백을 조정할 수 있을 것 같아요.

```suggestion
        let paramData: [String: String] = ["name": productNameTextField.text ?? "",
```